forcefield_utilities/gmso_xml.py

```diff
@@ -838,6 +855,64 @@ def load_from_etree(cls, root, existing):
                 children.append(pptype)
         return cls(children=children, **attribs)
 
+    def to_gmso_potentials(self, default_units):
+        potentials = {"pairpotential_types": {}}
+        parameters_units = filter(
+            lambda c: isinstance(c, ParametersUnitDef), self.children
+        )
+        units = {
+            parameter_unit.parameter: u.Unit(parameter_unit.unit)
+            for parameter_unit in parameters_units
+        }
+
+        for pairpotential_type in filter(
+            lambda c: isinstance(c, PairPotentialType), self.children
+        ):
+            pairpotential_type_dict = pairpotential_type.dict(
+                by_alias=True,
+                exclude={"children", "type1", "type2", "class1", "class2"},
+                exclude_none=True,
+            )
+
+            if "expression" not in pairpotential_type_dict:
+                pairpotential_type_dict["expression"] = self.expression
+
+            if pairpotential_type.type1 and pairpotential_type.type2:
+                pairpotential_type_dict["member_types"] = (
+                    pairpotential_type.type1,
+                    pairpotential_type.type2,
+                )
+
+            elif pairpotential_type.class1 and pairpotential_type.class2:
+                pairpotential_type_dict["member_classes"] = (
+                    pairpotential_type.class1,
+                    pairpotential_type.class2,
+                )
+
+            pairpotential_type_dict[
+                "parameters"
+            ] = pairpotential_type.parameters(units)
+            pairpotential_type_dict["independent_variables"] = indep_vars(
+                pairpotential_type_dict["expression"],
+                frozenset(pairpotential_type_dict["parameters"]),
+            )
+
+            gmso_pairpotential_type = GMSOBondType(**pairpotential_type_dict)
```

Comment:
shouldn't we use GMSOPairPotentialType? (https://github.com/mosdef-hub/gmso/blob/main/gmso/core/pairpotential_type.py)